modules/ROOT/pages/advlist.adoc

```diff
@@ -1,13 +1,17 @@
-= Advanced List plugin
-:navtitle: Advanced List
+= List Styles plugin
+:navtitle: List Styles
 :description: Create styled number and bulleted lists.
 :keywords: advlist, advlist_bullet_styles, advlist_number_styles
+:pluginname: List Styles
```

Comment:
Do we need to add anything to the keywords section just above?

Reply:
Good pickup, ill update